src/main/java/eu/europeana/metis/sandbox/service/record/RecordTierCalculationServiceImpl.java

```diff
@@ -56,12 +56,12 @@ public RecordTierCalculationView calculateTiers(RecordIdType recordIdType, Strin
         processingErrors.add(new ProcessingError(recordErrorLogEntity.getMessage(), recordErrorLogEntity.getStackTrace()));
       }
 
-      final String portalPublishRecordUrl = new UriTemplate(this.portalPublishRecordBaseUrl).expand(recordLog.getEuropeanaId())
+      final String portalPublishRecordUrl = new UriTemplate(this.portalPublishRecordBaseUrl).expand(recordLog.getRecordId().getEuropeanaId())
                                                                                             .toString();
       final String providerRecordUrl = new UriTemplate(this.providerRecordUrlTemplate).expand(datasetId, recordId, recordIdType)
                                                                                       .toString();
       final RecordTierCalculationViewGenerator recordTierCalculationViewGenerator = new RecordTierCalculationViewGenerator(
-          recordLog.getEuropeanaId(), recordLog.getRecordId(), recordLog.getContent(), portalPublishRecordUrl,
```

Comment:
Update branch with latest changes, I see this class was updated has less parameters. 
```
RecordTierCalculationViewGenerator
```

Reply:
I will handle this when merging the `develop` branch into this one. `develop` branch already has these changes